features/steps/room_steps.js

```diff
@@ -1,9 +1,15 @@
 const { Given, When, Then } = require("cucumber");
+const WorkspacePage = require("../page-objects/WorkspacePage");
+const Workspace = require("../parameter-types/workspaces");
+const Room = require("../parameter-types/rooms");
 const assert = require('assert').strict;
 
-Given("a Workspace with {accessLevel} {room}", function (accessLevel, room) {
-  // Write code here that turns the phrase above into concrete actions
-  return "pending";
+Given("a Workspace with {accessLevel} {room}", async function (accessLevel, room) {
+  const workspace = new Workspace("System Test");
+  this.workspace = new WorkspacePage(this.driver, workspace);
```

Comment:
Not blocking, something to think about is we have a WorkspacePage and a Workspace object and I'm not sure where the boundaries between these two things are. Does it make sense for there to be two different classes? Or could we do something to add the `driver` to the `Workspace` class? I'm not really sure. Something to noodle on over the next few months.